recipe/meta.yaml

```diff
@@ -40,6 +40,8 @@ requirements:
     - scipy >=1.8  # [aarch64]
     - joblib >=1.2.0
     - threadpoolctl >=3.1.0
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
```

Comment:
The same specifications are present in the `build` section.

Should it be removed for the current on that you propose?

Reply:
@xhochy: why mustn't the specification line 28 and line 29 be removed?

Reply:
Yes! In the perfect world, they would suffice.

Reply:
Could we just add the desired pin in `build`?

Should add I'm confused as to why the requirement in `build` is not already aligned with the compiler, which it is installed next to

Reply:
I outlined the issue in https://github.com/conda-forge/openmp-feedstock/issues/126 We actually install the desired pin in `build` but due to conda-builds internal behaviour, the run_exports is still taken from host. Thus, I don't see a solution where we don't add anything to host.

Reply:
Ah ok. Thank you Uwe! Very helpful 🙂

Wrote up a reply there on what we could consider next: https://github.com/conda-forge/openmp-feedstock/issues/126#issuecomment-2126480232

Agree this is the best we can do for now